### ISSUE TITLE
Object drop-down list doesn't disappear after selecting an object

### ISSUE THREAD

**Issue by BWPanda** [4ian/GDevelop#859]
## Describe the bug
When you click in a 'Choose an object' field, a drop-down list of available objects appears. When you select an object from this list however, the list doesn't disappear. Instead, all the other objects are removed and you're left with a very unhelpful list of the object you just chose. In addition, this often blocks you from being able to click on the next field under it.

## To Reproduce
See example:
![peek 2019-01-09 21-11](https://user-images.githubusercontent.com/2385329/50922581-8c49d200-1453-11e9-9d57-b15f065f7310.gif)

**Comment by 4ian** [comment]
Yes that's something that was added but that could be reverted. The only downside is that in this case, the first field is not focused, so that you can't use the modal with the keyboard alone. See https://github.com/4ian/GDevelop/pull/675

@blurymind if you want to re-make your PR, I think we can remove this for now unless we have a better way of focusing without opening the dropdown.

**Comment by blurymind** [comment]
That was fast :) thank you for fixing it

**Comment by 4ian** [comment]
Changes were simple: https://github.com/4ian/GDevelop/commit/56afb11e2c4abea98b82fb59fc248f80d6245403

but required to think a bit about all the different interactions to know the precise condition for having the dropdown opening.
